frontend/.env.example

```diff
@@ -120,4 +120,8 @@ MOCK_AUTH_ALLOWED_REDIRECTS=http://localhost:3000/auth/callback/raoidc
 SCCH_BASE_URI=http://www.example.com
 
 # Community where the documents were generated from, and user belongs. This value is linked to the specific vault database for the given community.
-CCT_VAULT_COMMUNITY='SecurityReview'
\ No newline at end of file
+CCT_VAULT_COMMUNITY='SecurityReview'
+
+
+# Memoization: amount of time in milliseconds before the cache will expire (undefined for infinity)
+GET_ALL_LETTER_TYPES_CACHE_TTL_SECONDS=0
```

Comment:
Milliseconds or seconds?

Reply:
should be milliseconds

Reply:
Do we need to specify milliseconds?

Reply:
@gregory-j-baker any comments?

Reply:
I feel like we don't need millisecond resolution for this, so make it easy on the person who might be configuring it and use seconds.. or even minutes, I guess.